script/testing/junit/src/FilterTrace.java

```diff
@@ -33,70 +31,80 @@ public static void main(String[] args) throws Throwable {
         Connection conn = db.getDbTest().newConn();
         List<String> tab = getAllExistingTableName(mog,conn);
         removeExistingTable(tab,conn);
+        List<String> aggregations = new ArrayList<>();
+        aggregations.add("COUNT");
+        aggregations.add("MAX");
+        aggregations.add("MIN");
+        aggregations.add("AVG");
+        aggregations.add("SUM");
```

Comment:
What is going on here? Why do we need this?

Reply:
We don't need this file for the PR. I was using FilterTrace to filter out traces from sqlite files.